src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown

```diff
@@ -35,234 +52,226 @@ Clears the console.
+Starts a new logging group with an optional title. All console output that
+occurs after `console.group()` and before `console.groupEnd()` is visually
+grouped together. 
 
 {% highlight js %}
-console.group("Authenticating user '%s'", user);
-console.log("User authenticated");
-console.groupEnd();
+function name(obj) {
+  console.group('name');
+  console.log('first: ', obj.first);
+  console.log('last: ', obj.last);
+  console.groupEnd();
+}
+
+name({"first":"wiley","last":"coyote"});
 {% endhighlight %}
 
-![Console group example](images/log-group-simple.png)
+![console.group() example](images/group.png)
 
 You can also nest groups:
 
 {% highlight js %}
-// New group for authentication:
-console.group("Authenticating user '%s'", user);
-// later...
-console.log("User authenticated", user);
-// A nested group for authorization:
-console.group("Authorizing user '%s'", user);
-console.log("User authorized");
-console.groupEnd();
-console.groupEnd();
+function name(obj) {
+  console.group('name');
+  console.log('first: ', obj.first);
+  console.log('last: ', obj.last);
+  console.groupEnd();
+}
+
+function doStuff() {
+  console.group('doStuff()');
+  name({"first":"wiley","last":"coyote"});
+  console.groupEnd();
+}
+
+doStuff();
 {% endhighlight %}
 
-![Nested logging group examples](images/nestedgroup-api.png)
+![nested console.group() example](images/nested-group.png)
 
-Learn more about [organizing console output](./console-write?#organizing-console-output).
+{% include shared/related_guides.liquid inline=true list=page.related-guides.organizing %}
 
 ## console.groupCollapsed(object[, object, ...])
 
-Creates a new logging group that is initially collapsed instead of open, as with `console.group()`.
+Creates a new logging group that is initially collapsed instead of open. 
 
 {% highlight js %}
-console.groupCollapsed("Authenticating user '%s'", user);
-console.log("User authenticated");
+console.groupCollapsed('status');
+console.log("peekaboo, you can't see me");
 console.groupEnd();
-console.log("A group-less log trace.");
 {% endhighlight %}
 
-![Creating a collapsed group](images/groupcollapsed.png)
+## console.groupEnd() {#groupend}
 
-## console.groupEnd()
-
-Closes the logging group that was most recently created with `console.group()` or `console.groupCollapsed()`. See `[console.group()](#consolegroupobject-object)` and `[console.groupCollapsed()](#consolegroupcollapsedobject-object)` for examples.
+Closes a logging group. See [`console.group`](#group) for an example.
 
 ## console.info(object [, object, ...])
 
-This method is identical in function to [`console.log()`](#consolelogobject--object-), but additionally shows an info icon next to the output.
-
-## console.log(object [, object, ...])
+Prints a message like [`console.log()`](#log) but also shows an info icon
+next to the output.
 
```

Comment:
You're leaving some things out. I assume the icon you're referring to is a triangle. That's a twisty lets you view the object's members. If any of the members are objects, it has twistys for those too.